ai/anthropic/v0/config/definition.json

```diff
@@ -0,0 +1,21 @@
+{
+  "availableTasks": [
+    "TASK_TEXT_GENERATION_CHAT"
+  ],
+  "custom": false,
+  "documentationUrl": "https://www.instill.tech/docs/latest/vdp/ai-connectors/anthropic",
+  "icon": "assets/anthropic.svg",
+  "icon_url": "",
```

Comment:
Important
`  "icon_url": ""`
Sorry, I miss this. Please take it out. 
We do not use it now.

Reply:
fixed